warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -0,0 +1,119 @@
+version: 2
+
+models:
+  - name: fct_vehicle_positions_messages
+    description: |
+      Each row is a message received from a vehicle positions GTFS RT feed.
+      See https://gtfs.org/realtime/reference/#message-vehicleposition for information
+      about message structure.
+    columns:
+      - name: key
+        description: |
+          Synthetic primary key constructed from `base64_url`, `extract_ts`, and entity `id`.
+        tests:
+          - unique:
+              #TODO: make an RT test macro that does this date subtraction
+              where: dt >= DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY)
+          - not_null:
+              where: dt >= DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY)
```

Comment:
we may need to revisit these depending on how many bytes these bill... maybe we also pick a specific hour of the day to eliminate more partitions?

Reply:
I forgot about this

Reply:
I'll filter to a rush hour hour Pacific